### PR TITLE
[vcpkg docs] Documenting VSLANG

### PR DESCRIPTION
**Pull request by JavierMatosD**
This PR documents the VSLANG environment variable.

### CODE REVIEW

**Review by JonLiu1993 — DISMISSED** [review]

**Review by BillyONeal — DISMISSED** [review]
Dropping reviewed: added @ras0219-msft and @AugP for final approval of docs change.

File: docs/users/config-environment.md [hunk 1]
```diff
@@ -103,3 +103,10 @@ This environment variable allows using a private mirror for all SHA512-tagged as
 #### VCPKG_NO_CI
 
 Setting `VCPKG_NO_CI` disables vcpkg's CI environment detection heuristics.
+
+#### VSLANG
+
+This environment variable allows us to change the language used to display messages. It should be set to one of the 14 supported LCIDs (locale identifier, 4-byte value corresponding to a language). 
+
+For example: 1033 corresponds to the English language. For a full list of support LCIDs see [Localization](https://github.com/microsoft/vcpkg-tool/blob/main/docs/localization.md)
```
Comment (AugP) [inline docs/users/config-environment.md:111]: A few minor comments: 
1. Is "1033" referring to English broadly, or US English? Are there different codes for US vs. UK English? 
2. Break the "For a full list of..." sentence into its own line. 
3. Minor typo: "Full a full list of support LCIDs". The "support" should be "supported". Also needs a period at the very end of the sentence.

File: docs/users/config-environment.md [hunk 2]
```diff
@@ -103,3 +103,10 @@ This environment variable allows using a private mirror for all SHA512-tagged as
 #### VCPKG_NO_CI
 
 Setting `VCPKG_NO_CI` disables vcpkg's CI environment detection heuristics.
+
+#### VSLANG
+
+This environment variable allows us to change the language used to display messages. It should be set to one of the 14 supported LCIDs (locale identifier, 4-byte value corresponding to a language).
```
Comment (AugP) [inline docs/users/config-environment.md:109]: I would rephrase to not use first person (us): "This environment variable sets the language vcpkg uses to display messages."

**Review by AugP — COMMENTED** [review]
Thanks, left a few comments.

**Comment by Cheney-W** [comment]
Close this PR and please track it with https://github.com/microsoft/vcpkg-docs/pull/33.
